src/middleware/middleware.js

```diff
@@ -0,0 +1,33 @@
+export const validateEmail = (email) => {
+  // eslint-disable-next-line no-useless-escape
+  const mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  return mailformat.test(email);
+};
+export const validPhoneNo = (number) => {
+  const phoneno = /^\d{10}$/;
+  return phoneno.test(number);
+};
+
+export const validateUserInput = (req, res, next) => {
+  const {
+    firstName, lastName, email,
+    password, phoneNo
+  } = req.body;
+
+  if (firstName.length < 3 || firstName === '') {
```

Comment:
Why do you need to check for empty string again `firstName === ''` ? if the value is empty then `firstName.length` will be equal to `0` which is also satisfied by the initial condition `firstName.length < 3` 👀  or am I missing something here? please update this in other validation also